src/genie/libs/parser/iosxe/tests/ShowApConfigGeneral/cli/equal/golden_output_expected.py

```diff
@@ -0,0 +1,115 @@
+expected_output = {
+    "ap_config_general_info": {
+        "bg-1-cap1": {
+            "cisco_ap_identifier": "70b3.d278.e03e",
+            "country_code": "IN",
+            "regulatory_domain_allowed_by_country": "802.11bg:-A   802.11a:-DN",
+            "ap_country_code": "IN  - India",
+            "slot_0": "-A",
+            "slot_1": "-D",
```

Comment:
respect an indentation in output. so, schema could be:
```
"ap_regulatory_domain": {
    "slot_0": "-A",
    "slot_1": "-D",

Reply:
Updated